.github/workflows/ci.yml

```diff
@@ -65,15 +76,15 @@ jobs:
             meson-python \
             "cypari2 >=2.2.1" \
             "cysignals >=1.11.2, != 1.12.0" \
-            "cython >=3.0, != 3.0.3" \
+            "cython >=3.0, != 3.0.3, < 3.1.0" \
```

Comment:
this starts looking like a repetition of the very same list of constraints. It certainly has a potential to become a nightmare to maintain. Can this be brought into something like a separate GH Action?